src/pages/account/logins/index.tsx

```diff
@@ -232,3 +249,24 @@ const Button = styled.button<ButtonProps>`
     color: ${({ theme }) => theme.colors.contrastCopy};
   }
 `;
+
+export const ShowMoreButton = styled.button<ButtonProps>`
+  padding: 0.5rem;
+  margin-right: 1rem;
```

Comment:
Use spacing values from the theme.

Reply:
We're not using theme anymore in this file